jsonnet/thanos-ruler.jsonnet

```diff
@@ -145,6 +145,26 @@ local authorizationRole =
         secret.mixin.metadata.withNamespace(thanosRulerConfig.namespace) +
         secret.mixin.metadata.withLabels({ 'k8s-app': 'thanos' }),
 
+      // https://thanos.io/components/rule.md/#configuration
+      alertmanagerConfigSecret: {
+        kind: 'Secret',
+        alertmanagers: [
+          {
+            http_config: {
+              bearer_token_file: '/var/run/secrets/kubernetes.io/serviceaccount/token',
+              tls_config: {
+                insecure_skip_verify: 'true',
```

Comment:
this should a bool?

https://github.com/thanos-io/thanos/blob/8a6746298ec39ccb40108e5517535e944d3920d3/pkg/http/http.go#L51